src/redux/categories/categories.js

```diff
@@ -0,0 +1,13 @@
+const CHECK_STATUS = 'book-store/categories/CHECK_STATUS';
+
+const categoryReducer = (state = [], action = {}) => {
+  if (action.type === CHECK_STATUS) {
+    return 'Under construction';
+  }
+  return state;
+};
+
+const checkStatus = (book) => ({ type: CHECK_STATUS, book });
```

Comment:
- [ ] Kindly be noticed that there is no need to make the `checkStatus` function take a book parameter.